Gemfile

```diff
@@ -79,6 +79,10 @@ gem 'bootstrap4-kaminari-views'
 gem 'devise', "~> 4.5" # user accounts and login
 gem 'access-granted', "~> 1.0" # authorization
 
+# decorating and truncating html
+gem "rails_autolink"
```

Comment:
Current rails_autolink version is `1.1.6`, so this should be `gem "rails_autolink", '~> 1.0'` to limit to `1.x` version (not that I expect a 2.0 to ever be released, it's pretty much abandoned, but this is good Gemfile hygeine). 

The only cases I don't do it are in some cases where gem only exists in pre 1.0, so it's unclear what the version specification should be, although even in those cases I'm leaning toward `'< 2.0'` in favor of entirely unrestricted.